igneous_cli/cli.py

```diff
@@ -1896,7 +1896,7 @@ def create(
     if arr.shape[0] == 3 and arr.ndim == 3:
       arr = arr[..., np.newaxis]
       arr = np.transpose(arr, axes=[1,2,3,0])
-  elif ext == ".nii":
+  elif ext == ".nii" or ext == ".nii.gz":
```

Comment:
After normalization, ext should be ".nii" I think.

Reply:
Indeed, if we get only .nii, we don't have to edit.
```suggestion
  elif ext == ".nii":
```